src/core/solvers/ddp.cpp

```diff
@@ -14,9 +14,10 @@ namespace crocoddyl {
 
 SolverDDP::SolverDDP(boost::shared_ptr<ShootingProblem> problem)
     : SolverAbstract(problem),
-      regfactor_(10.),
-      regmin_(1e-9),
-      regmax_(1e9),
+      reg_incfactor_(10.),
+      reg_decfactor_(10.),
+      reg_min_(1e-16),
+      reg_max_(1e12),
```

Comment:
This part should follow the changes from https://github.com/loco-3d/crocoddyl/pull/893#discussion_r542122288